jobs/build/merge_ocp/Jenkinsfile

```diff
@@ -49,7 +49,7 @@ node {
 
     SSH_KEY_ID = "openshift-bot"
     MERGE_VERSIONS = VERSIONS.split(',')
-    CURRENT_MASTER = "4.0"
+    CURRENT_MASTER = "4.1"
```

Comment:
I think we should look at moving this kind of value into `commonlib` or something equivalent.